packages/react-components/src/components/table/table.tsx

```diff
@@ -213,9 +223,8 @@ export const Table = <T extends Record<string, unknown>>({
     visibleColumns,
     setPageSize,
     setHiddenColumns,
-    selectedFlatRows,
     state: {
-      pageSize, pageIndex, sortBy
+      pageSize, pageIndex, sortBy, selectedRowIds: sel
```

Comment:
maybe we can find a better and explicative name instead of `sel`

Reply:
What do you think about activeSelectedRowIds?

Reply:
First of all, why we are renaming `selectedRowIds` if they are the same?

Reply:
`selectedRowIdsState`